src/crypto/index.ts

```diff
@@ -576,6 +577,18 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
         }
     }
 
+    public supportsSecretsForQrLogin(): boolean {
+        return false;
+    }
+
+    public async exportSecretsForQrLogin(): Promise<QRSecretsBundle> {
+        throw new Error("Method not implemented.");
```

Comment:
We seem to use `Not implemented` already so this is consistent.

![image](https://github.com/matrix-org/matrix-js-sdk/assets/2403652/48913770-527c-4a48-83b6-2bb6c1e554a2)